Makefile

```diff
@@ -50,7 +52,8 @@ BROKER_DOCKER_OPTS=--rm -v $(PAK_CACHE):$(PAK_CACHE) -v $(PWD):/brokerpak -w /br
 		-e "DB_TYPE=sqlite3" \
 		-e "DB_PATH=/tmp/csb-db" \
 		-e PAK_BUILD_CACHE_PATH=$(PAK_CACHE) \
-		-e GSB_PROVISION_DEFAULTS
+		-e GSB_PROVISION_DEFAULTS \
+		-e GSB_COMPATIBILITY_ENABLE_BETA_SERVICES=$(GSB_COMPATIBILITY_ENABLE_BETA_SERVICES)
```

Comment:
thoughts (no-blocking): If we name an environment variable without specifying a value, then the current value of the named variable is propagated into the container's environment:

`
$ export today=Wednesday
$ docker run -e "deep=purple" -e today --rm alpine env
`
![image](https://user-images.githubusercontent.com/9041034/175312475-821069e1-3085-436f-ac1c-a45799cad5cb.png)

Reply:
done